api/src/main/java/org/bahmni/module/events/model/Event.java

```diff
@@ -0,0 +1,28 @@
+package org.bahmni.module.events.model;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class Event {
+	
+	public final String eventId;
+	
+	public final String eventType;
```

Comment:
Why is eventType a string, instead of a type?

Reply:
Replaced eventType with new enum as BhamniEventType